webrtc-stats.html

```diff
@@ -1341,13 +1341,14 @@ <h2>
               </dd>
               <dt>
                 <dfn><code>totalInterFrameDelaySquared</code></dfn> of type <span class=
-                "idlMemberType">unsigned long long</span>
+                "idlMemberType">double</span>
               </dt>
               <dd>
                 <p>
-                  Sum of the squared interframe delay in ms between consecutive frames. The
-                  interframe variance can be calculated from this value and <a>framesDecoded</a>
-                  at two timestamps <code>t0</code> and <code>t1</code>.
+                  Sum of the squared interframe delay in seconds between consecutive frames. The
```

Comment:
When you say "between two frames", what time are you referring to with regards to a frame? Make it clear that "a frame" means when a frame is decoded, i.e. at the same time that framesDecoded increments.

I.e. if the measurement was performed just _before_ decoding then the calculations dividing by framesDecoded would not work.

Reply:
Good point, will fix.

Reply:
The more I think of it, it's a bit tricky to do the conversion to standard deviation since the timestamp of when the last frame was decoded is not available.
I made an attempt to include the text and still be correct about it being an approximation.

Reply:
Adding totalInterframeDelay has made the complicated text unnecessary.